compute/ARMComputeEx/src/runtime/CL/functions/CLReduceOperation.cpp

```diff
@@ -59,6 +59,8 @@ Status CLReduceOperation::validate(const ITensorInfo *input, const ITensorInfo *
   const size_t num_of_kernels = axis.size();
   const size_t num_of_interm_tensors = num_of_kernels - (keep_dims ? 1 : 0);
 
+  ARM_COMPUTE_RETURN_ERROR_ON(num_of_kernels < 1);
```

Comment:
@hseok-oh is `axis.size() == 0` not allowed?

Reply:
`axis.size() == 0` means there is no axis to reduce. And it makes invalid result to calculate `num_of_interm_tensors` and invalid access to `interm_tensors` array.